cdc/changefeed.go

```diff
@@ -845,3 +875,47 @@ func (c *changeFeed) pullDDLJob() error {
 	}
 	return nil
 }
+
+// startSyncPeriod start a timer for every changefeed to create sync point by time
+func (c *changeFeed) startSyncPeriod() {
+	//c.startTimer <- true
+	go func() {
+		for {
+			//select {
+			//case <-c.startTimer:
+			time.Sleep(syncInterval)
+			c.updateResolvedTs = false
+			//}
+		}
+	}()
+}
+
+//createSynctable create a sync table to record the
+func (c *changeFeed) createSynctable() {
+	database := "TiCDC"
+	c.syncDB.Exec("CREATE DATABASE IF NOT EXISTS " + database)
+	c.syncDB.Exec("USE " + database)
+	c.syncDB.Exec("CREATE TABLE  IF NOT EXISTS syncpoint ( master_ts varchar(18),slave_ts varchar(18),PRIMARY KEY ( `master_ts` ) )")
```

Comment:
Please use `primary_ts` and `secondary_ts`, since `mas***` and `sla***` are controversial

Reply:
Should we add more fileds in the `syncpoint` table, such as a column to record `changefeed id`, since we could create multiple changefeeds to replicate to the same downstream database, such as 
- changefeed1: db1 -> db1
- changefeed2: db2 -> db2